front/SettingsForm.tsx

```diff
@@ -88,6 +90,7 @@ const SettingsForm = () => {
           <fieldset className="form-group">
             <input
               className="form-control form-control-lg"
+              data-cy="updateEmailInput"
```

Comment:
The `data-cy` attribute for the `updateEmailInput` input field is correctly added, which is important for Cypress testing.